index.md

```diff
@@ -13,107 +12,166 @@ First we set up & capture the content, then we render it in the scaffolding belo
 
 
 {% capture intro-left %}
-## ![Cockpit](/images/site/cockpit-logo.svg)
-{:.logo}
-
-<span aria-hidden="true" role="presentation">Cockpit</span> is a server manager that makes it easy to administer your GNU/Linux servers via a web browser.
-
+Cockpit makes it easy to administer your GNU/Linux servers via a web browser.
 {% endcapture %}
 
 {% capture intro-right %}
 [![Docker screenshot]({{ site.baseurl }}/images/site/screenshot-docker.png)]({{ site.baseurl }}/images/site/screenshot-docker.png){:.screenshot.zoom}
 {% endcapture %}
 
 
-{% capture blurb_easy %}
-[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png){:.screenshot.zoom}
-### Easy to use
+{% assign blurbs_highlight = 6 %}
+
+{% capture blurbs %}
+
+## Easy to use
+
+### Discoverable
 Cockpit makes Linux discoverable, allowing sysadmins to easily perform tasks such as starting containers, storage administration, network configuration, inspecting logs and so on.
-{% endcapture %}
 
+[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png)
+{:.screenshot.zoom}
+
+### Designed & usability-tested
+Cockpit has been designed with your goals in mind. We also routinely test Cockpit with usability studies to make it work the way you'd expect. As a result, Cockpit gets easier to use all the time.
+
+### Team friendly
+Cockpit is useful for seasoned admins and is friendly enough for those new to Linux. We've designed Cockpit to be friendly for whole teams.
 
-{% capture blurb_interactive %}
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
+
+---
+
+## Integrated
+
+### Plays well with others
 Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
-{% endcapture %}
 
+[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png)
+{:.screenshot.zoom}
+
+### Sign in like normal
+By default, Cockpit uses [system credentials]({{ site.baseurl }}/guide/latest/privileges). You don't need to set up any special accounts. Network-wide logins are also supported through [single-sign-on]({{ site.baseurl }}/guide/latest/sso) and other [authentication]({{ site.baseurl }}/guide/latest/authentication) techniques.
```

Comment:
I would push a little bit extra about the fact that system credentials are respected.
A lot of the questions I've gotten from customers revolve around the fact that they want to make sure admins with less privileges can't do stuff to damage the system.